### PR TITLE
[#24] Prepare for announcement

### PR DESCRIPTION
**Pull request by rexolion**
## Related issue(s)

Resolves: #24 

#### Related changes (conditional)

- Tests
  - [ ] If I added new functionality, I added tests covering it.
  - [ ] If I fixed a bug, I added a regression test to prevent the bug from silently reappearing again.

- Documentation
  - [ ] I checked whether I should update the docs and did so if necessary:
    - [README](../tree/master/README.md)
  - [ ] I checked whether I should update/add links to the figma UI-kit and did so if necessary:
    - [FIGMA UI-KIT](https://www.figma.com/file/2St3zSul4fHnLffqy3WK7P/%5B-union-%5D-mobile?node-id=4687%3A34299)

- Public contracts
  - [ ] My changes doesn't break any Licenses.
  - [ ] I added an entry to the [changelog](../tree/master/CHANGES.md) if my changes are visible to the users
        and
  - [ ] provided a migration guide for breaking changes if possible

#### Stylistic guide (mandatory)

- [ ] My commits comply with [the policy used in Union](https://github.com/union-platform/guides/blob/master/commits.md).
- [ ] My code complies with the [styles guide](https://github.com/union-platform/guides/blob/master/README.md#programming-languages).

#### Before merging (mandatory)
- [ ] Check for `fixup!` or `squash!` commits in this branch
- [x] Check __target__  branch of PR is set correctly

### CODE REVIEW

**Review by rexolion — COMMENTED** [review]
| Device | URL |
|--|--|
| mobile | https://www.unionapp.cc |

Not what you expected? Are your scores flaky? **GitHub runners could be the cause.**
[Try running on Foo instead](https://www.foo.software/docs/lighthouse-check-github-action/examples#running-on-foo-and-saving-results)

File: src/_data/footer.json [hunk 1]
```diff
@@ -1,51 +1,36 @@
...
     {
-      "link":"http://ui.unionapp.cc/",
-      "text":"UI-Library"
+      "link": "https://github.com/union-platform",
+      "text": "Github"
     },
     {
-      "link":"mailto:",
-      "text":"artyom@unionapp.cc"
+      "link": "https://t.me/unionapp_cc_dev",
```
Comment (worm2fed) [inline src/_data/footer.json:14]: sorry, I've changed chat link

https://t.me/unionapp_cc_discussions

File: src/_data/nav.json [hunk 2]
```diff
@@ -1,6 +1,9 @@
...
-  "navLogo":"/assets/images/unionLogo.svg",
-  "navLinks":["home","ui-library","github"],
-  "navBtnLink":"mailto:artyom@unionapp.cc",
-  "navBtnText":"Get in touch"
+  "navLogo": "/assets/images/unionLogo.svg",
+  "navLinks": [
+    "ui-library",
+    "github"
+  ],
+  "navBtnLink": "https://t.me/unionapp_cc_dev",
```
Comment (worm2fed) [inline src/_data/nav.json:7]: sorry, I've changed chat link

https://t.me/unionapp_cc_discussions

File: src/index.md [hunk 3]
```diff
@@ -4,14 +4,14 @@ hero:
   title:
     firstPara: "We unite people"
     secondPara: "to achieve common goals"
-    subhead: "Find an activity you like, join an interesting project or create your own."
+    subhead: "Find your labor of love, join an interesting project or create your own."
   connectBtn:
     text: "Contact us"
-    link: "mailto:artyom@unionapp.cc"
+    link: "https://t.me/unionapp_cc_dev"
```
Comment (worm2fed) [inline src/index.md:10]: sorry, I've changed chat link

https://t.me/unionapp_cc_discussions

**Review by worm2fed — CHANGES_REQUESTED** [review]

**Comment by worm2fed** [comment]
@rexolion maybe we can have single source for same links across all files?

**Comment by rexolion** [comment]
> @rexolion maybe we can have single source for same links across all files?

Great idea, but I think for now it's ok. When the website would become bigger, it will be more sense in it.

**Comment by worm2fed** [comment]
good!

>  Fix white vertical lines in animation
are you going to fix it? or postpone?

**Comment by rexolion** [comment]
> good!
> 
> > Fix white vertical lines in animation
> 
> are you going to fix it? or postpone?

Yeah, fuck it. It's easier to remove it or ignore it than fix it — weird behavior of SVG animations.

**Comment by worm2fed** [comment]
@rexolion in chrome it's fine, but not in safari

did you saw this? https://stackoverflow.com/questions/37726195/why-does-my-css-animation-not-work-in-safari

pls try; if it will not work - lets proceed without it

File: src/_data/footer.json [hunk 4]
```diff
@@ -1,51 +1,36 @@
...
     {
-      "link":"http://ui.unionapp.cc/",
-      "text":"UI-Library"
+      "link": "https://github.com/union-platform",
+      "text": "Github"
     },
     {
-      "link":"mailto:",
-      "text":"artyom@unionapp.cc"
+      "link": "https://t.me/unionapp_cc_discussions",
```
Comment (worm2fed) [inline src/_data/footer.json:14]: https://t.me/unionapp_cc

File: src/_data/footer.json [hunk 5]
```diff
@@ -1,51 +1,36 @@
...
+      "text": "Telegram Chat"
```
Comment (worm2fed) [inline src/_data/footer.json:15]: Telegram

File: src/_data/footer.json [hunk 6]
```diff
@@ -1,51 +1,36 @@
...
-    },
-    {
-      "link":"#",
-      "text":"Dribbble"
-    },
-    {
-      "link":"#",
-      "text":"Instagram"
+      "link": "https://t.me/unionapp_cc",
+      "text": "Telegram Channel"
```
Comment (worm2fed) [inline src/_data/footer.json:33]: Telegram

File: src/_data/nav.json [hunk 7]
```diff
@@ -1,6 +1,9 @@
...
-  "navBtnLink":"mailto:artyom@unionapp.cc",
-  "navBtnText":"Get in touch"
+  "navLogo": "/assets/images/unionLogo.svg",
+  "navLinks": [
+    "ui-library",
+    "github"
+  ],
+  "navBtnLink": "https://t.me/unionapp_cc_discussions",
+  "navBtnText": "Get in touch"
 }
```
Comment (worm2fed) [inline src/_data/nav.json:9]: Missing newline at the end of file

File: src/_includes/layouts/features.njk [hunk 8]
```diff
@@ -7,65 +7,72 @@
...
+              </li>
+            {% endif %}
+          </ul>
+        </nav>
+      </div>
+    </div>
+  </div>
+</div>
+</section>
 {% endblock %}
```
Comment (worm2fed) [inline src/_includes/layouts/features.njk:78]: Missing newline at the end of file

File: src/_includes/layouts/homepage.njk [hunk 9]
```diff
@@ -62,47 +62,53 @@
...
+              {{commits.des}}
+            </p>
           </div>
         </div>
       </div>
-    </section>
+    </div>
+  </section>
 
 {% endblock %}
```
Comment (worm2fed) [inline src/_includes/layouts/homepage.njk:114]: Missing newline at the end of file

File: src/features/activities.md [hunk 10]
```diff
@@ -47,4 +47,4 @@ Mystery answer. After entering this mode, the screen initially displays “ask n
 
 Note: these strings are for the English version of the watch, we will need to use completely different strings in other countries without reprogramming the logic of the watch.
                 
-The user may cycle among all modes (date/time, timer, mystery answer) by pressing the mode button.     
\ No newline at end of file
+The user may cycle among all modes (date/time, timer, mystery answer) by pressing the mode button.
```
Comment (worm2fed) [inline src/features/activities.md:50]: Missing newline at the end of file

File: src/features/people.md [hunk 11]
```diff
@@ -47,4 +47,4 @@ Mystery answer. After entering this mode, the screen initially displays “ask n
 
 Note: these strings are for the English version of the watch, we will need to use completely different strings in other countries without reprogramming the logic of the watch.
                 
-The user may cycle among all modes (date/time, timer, mystery answer) by pressing the mode button.
\ No newline at end of file
+The user may cycle among all modes (date/time, timer, mystery answer) by pressing the mode button.
```
Comment (worm2fed) [inline src/features/people.md:50]: Missing newline at the end of file

**Review by worm2fed — CHANGES_REQUESTED** [review]

File: src/_data/nav.json (hunk 7 above)
Comment (rexolion) [reply src/_data/nav.json:9]: We don't put a newline on EOF in JSON format. It's a data format.

Comment (rexolion) [reply src/_data/nav.json:9]: But fuck it, I made it :)

Comment (worm2fed) [reply src/_data/nav.json:9]: It can be treated as config as well. Any way it's source file as for me)

Thank you))

File: .gitignore [hunk 12]
```diff
@@ -6,4 +6,7 @@ node_modules
 _site
 dist
 package-lock.json
-yarn.lock
\ No newline at end of file
+yarn.lock
+.yarn
+.pnp.cjs
+.pnp.loader.mjs
```
Comment (worm2fed) [inline .gitignore:12]: And newline here as well))

**Comment by rexolion** [comment]
@worm2fed  Do you want Telegram links to be this way?:

image

Or every link in contacts?

**Comment by worm2fed** [comment]
@rexolion lets hide titles in footer for now) and I think telegram channel is enough (without chat)

**Review by worm2fed — APPROVED** [review]

**Pull request merged**